docs/de/docs/advanced/additional-status-codes.md

```diff
@@ -0,0 +1,69 @@
+# Zusätzliche Statuscodes
+
+Standardmäßig gibt **FastAPI** die Responses als `JSONResponse` zurück und fügt den Inhalt, den Sie von Ihrer *Pfadoperation* zurückgeben, in diese `JSONResponse` ein.
```

Comment:
Würde ich geringfügig umschreiben, damit es einfacher verständlich ist: Standardmäßig liefert **FastAPI** die Rückgabewerte (Responses) als `JSONResponse` zurück und fügt den Inhalt der jeweiligen *Pfadoperation* in das `JSONResponse` Objekt ein.